pkgs/applications/virtualization/catatonit/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, glibc }:
+
+stdenv.mkDerivation rec {
+  version = "0.1.5";
+  pname = "catatonit";
+
+  src = fetchFromGitHub {
+    owner = "openSUSE";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-ciJ1MI7jr5P2PgxIykQ+BiwNUO8lQHGt0+U8CNbc5bI=";
+  };
+
+  patches = [
+    # Fix compilation with musl
+    (fetchpatch {
+      url = "https://github.com/openSUSE/catatonit/commit/75014b1c3099245b7d0f44f24d7f6dc4888a45fd.patch";
+      sha256 = "sha256-9VMNUT1U90ocjvE7EXYfLxuodDwTXXHYg89qqa5Jq0g=";
+    })
+  ];
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = lib.optionals (!stdenv.hostPlatform.isMusl) [ glibc glibc.static ];
```

Comment:
I don't think you need `glibc.static` explicitly.

Reply:
Without `glibc.static` the executable would be linked to `/nix/store/.../libc.so.6` dynamically (checked it). We want a fully static binary so it can work inside any container.

Reply:
I find that a bit odd but I'll take your word for it. Thanks for humoring me :)